src/components/Tag/Tag.tsx

```diff
@@ -19,7 +19,11 @@ export const Tag = (
   const tagClasses = classnames('usa-tag', className)
 
   return (
-    <span data-testid="tag" className={tagClasses} style={{ ...style }}>
+    <span
+      data-testid="tag"
+      className={tagClasses}
+      style={{ ...style }}
+      {...spanProps}>
```

Comment:
Thanks for catching this, I think I saw a couple other places where this rewrite was kind of a cleanup for us.